src/mono/mono.proj

```diff
@@ -2,6 +2,8 @@
   <Import Project="Directory.Build.props" />
   <Import Project="Directory.Build.targets" />
 
+  <!-- Foo -->
```

Comment:
The OCD part of my brain is upset that you choose to capitalize Foo differently in different files.

Reply:
hahah 😄